codalab/lib/metadata_util.py

```diff
@@ -25,7 +25,12 @@ def fill_missing_metadata(bundle_subclass, args, initial_metadata):
     '''
     # Fill in default values for all unsupplied metadata keys.
     new_initial_metadata = {}
-    for spec in bundle_subclass.get_user_defined_metadata():
+    for spec in bundle_subclass.METADATA_SPECS:
```

Comment:
What's the motivation for this change?

Reply:
There were certain metadata fields that weren't getting returned to the frontend because the bundle didn't have a value set for that field. E.g. there is no `data_size` util the bundle is in the `running` state. Since the metadata descriptions are pulled from the backend, this resulted in some fields not having a tooltip / description until that field was set for the bundle.

The fix that I came up with was to set default values for those missing fields so that they're always returned. This change enables that.